stackoverflow-dark.css

```diff
@@ -88,8 +88,8 @@
 
   /* === background color === */
   /* from darkest to lightest grey */
-  .search-container input[type=text] {
-    background-color: #000 !important;
+  .search-container input[type=text], .topbar.newheader .topbar-wrapper .topbar-links .search-container #search input[name="q"] { 
+    background-color: #222 !important;
```

Comment:
Search looked out of place if it was black.
I'm guessing they are trying out something, or could be that I'm not logged in. (I didn't opt in anywhere)
![image](https://cloud.githubusercontent.com/assets/16362320/21080416/fcd821f8-bfae-11e6-9d95-63bbf685a7a7.png)